exercises/word-search/canonical-data.json

```diff
@@ -0,0 +1,593 @@
+{
+  "exercise": "word-search",
+  "version": "1.0.0",
+  "comments": [
+    "Grid rows and columns are 1-indexed."
+  ],
+  "cases": [
+    {
+      "description": "Should locate words written left to right",
+      "property": "search",
+      "grid": [
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "screeaumgr",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
+      ],
+      "wordsToSearchFor": [
+        "clojure"
+      ],
+      "expected": [
+        {
+          "start": {
+            "column": 1,
+            "row": 10
+          },
+          "end": {
+            "column": 7,
+            "row": 10
+          }
+        }
+      ]
+    },
+    {
+      "description": "Should locate words written right to left",
+      "property": "search",
+      "grid": [
+        "jefblpepre",
+        "camdcimgtc",
+        "oivokprjsm",
+        "pbwasqroua",
+        "rixilelhrs",
+        "wolcqlirpc",
+        "screeaumgr",
+        "alxhpburyi",
+        "jalaycalmp",
+        "clojurermt"
```

Comment:
Indeed - before the schema, we had repeated inputs for`tournament` and `pov`. The schema did not support that, so in #699 we discussed and decided not to add the support.